api/composer.json

```diff
@@ -56,7 +56,7 @@
     "scripts": {
         "auto-scripts": {
             "cache:clear": "symfony-cmd",
-            "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
+            "assets:install %PUBLIC_DIR%": "symfony-cmd",
             "security-checker security:check": "script"
```

Comment:
I noticed that the Pull Request #24 has removed the command `bin/console assets:install` https://github.com/api-platform/demo/pull/24/files#diff-507acfb69e262d39c47cc1a687c1aab4L12 and symbolic links prevent assets loading.

The flex recipe remove now "--relative" and "--symlink" options by default   See https://github.com/symfony/recipes/pull/406